### ISSUE TITLE
`#[oai(example)]` causes compilation errors with generated structs.

### ISSUE THREAD

**Issue by wvffle** [NexRX/restructed#3]
Hey there, restructed seems to be a really great utility. 

When testing it out, I saw that this will pass down the `#[oai(example)]` to all generated structs causing compilation errors as those do not have the implementation of Example trait.

```rust
use restructed::Models;

#[derive(poem_openapi::Object, Models)]
#[oai(example, skip_serializing_if_is_none, rename_all = "camelCase")] // <-- this here
#[model(base(derive(poem_openapi::Object, Debug)), defaults(preset = "read"))]
#[patch(UserUpdate, preset = "write")]
pub struct User {
    #[oai(read_only)]
    pub id: u32,

    #[oai(validator(min_length = 3, max_length = 16, pattern = r"^[a-zA-Z0-9_]*$"))] // oai attributes carry over with `preset = write/write` or attributes_with="oai"
    pub username: String,

    #[oai(validator(min_length = 5, max_length = 1024), write_only)]
    pub password: String,
}
```

Maybe it would be possible to automatically implement this trait based on the original struct's implementation?

**Comment by NexRX** [comment]
Thanks for the issue and feedback! 😊 And appoligies for not getting back as quick as I'd like.

I agree with the suggestion since this crate does explicitly support `#[oai]` attributes. It would be possible to automatically handle this by calling the parents implementation of `trait Example` and then using already implemented `trait From` implementation. Not at all hard to-do so expect a release today soon.

**Comment by NexRX** [comment]
Thanks @wvffle for your contributed feedback for this enhancement. Great to see it and if you have any more, please do share ❤️ 

Closed with the release of [v0.2.1](https://github.com/NexRX/restructed/releases/tag/v0.2.1) now available via crates.io

**Comment by wvffle** [comment]
Thanks for quick reaction!

If I find any bugs or encounter any more issues, I will surely report them :)
For now everything seems to work fine.

Kind regards
